core/App/navigators/ProofRequestStack.tsx

```diff
@@ -17,6 +18,13 @@ const ProofRequestStack: React.FC = () => {
   const defaultStackOptions = createDefaultStackOptions(theme)
   return (
     <Stack.Navigator screenOptions={{ ...defaultStackOptions }}>
+      <Stack.Screen
+        name={Screens.ProofRequestFullName}
```

Comment:
Let's call this screen and component `ProofRequestDetails`

Reply:
done